example-app/SantokuApp/src/apps/app/navigators/AuthenticatedStackNav.tsx

```diff
@@ -65,6 +65,19 @@ const Component: React.FC<Props> = ({initialData}) => {
       <nav.Screen
         name="QuestionAndEventStackNav"
         component={QuestionAndEventPostStackNav}
+        /*
+         presentationに「modal」「transparentModal」「containedModal」「containedTransparentModal」「formSheet」を指定した場合は、
+         端末の向きを変更（横画面⇔縦画面）しても画面の向きは変わりません。
```

Comment:
```suggestion
         端末の向きを変更（横画面⇔縦画面）しても画面の向きが変わらない場合があります。
```

すぐ下に `【発生環境】` とあるが、ここだけ読むと 100%発生するように読めるため。